client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -167,7 +168,7 @@ const LoggedOutForm = React.createClass( {
 		return (
 			<LoggedOutFormLinks>
 				<LoggedOutFormLinkItem href={ login( { redirectTo } ) }>
-					{ this.translate( 'Already have an account? Sign in' ) }
+					{ i18n.translate( 'Already have an account? Sign in' ) }
```

Comment:
@aduth Thanks for that information. I'm in the process of breaking down #12991 so I'm trying to find a small change set. I'll need to make the es6 class changes first so I can work with HOCs then revisit this.

Changing to `[Status] In Progress`.